### PR TITLE
[exporter/postgres]: Exporter to OTEL data in Postgres

### PR DESCRIPTION
**Pull request by destrex271**
**Description:**
Initial Draft PR for a Postgres Exporter, will update it with the folder structure soon

**Link to tracking Issue:**  
 - #35451

### CODE REVIEW

File: exporter/postgresexporter/config.go [hunk 1]
```diff
@@ -0,0 +1,20 @@
+package postgresexporter
```
Comment (atoulme) [inline exporter/postgresexporter/config.go:1]: headers

**Comment by atoulme** [comment]
You need a metadata.yaml file and generate the README and most from there. Your initial stage should be in development, not alpha. Your initial PR should have the skeleton of the component, logic is added afterwards.

But most important, please secure a sponsor first.

**Comment by philippemnoel** [comment]
> You need a metadata.yaml file and generate the README and most from there. Your initial stage should be in development, not alpha. Your initial PR should have the skeleton of the component, logic is added afterwards.
> 
> But most important, please secure a sponsor first.

What does securing a sponsor entail? I'm the CEO of ParadeDB (paradedb.com). We're happy to push for this feature, Akshat is one of our contributors and has been putting this PR as part of his contributions to @paradedb).

File: exporter/postgresexporter/config.go (hunk 1 above)
Comment (destrex271) [reply exporter/postgresexporter/config.go:1]: Okk, will do that.

**Comment by destrex271** [comment]
I think this branch has become a huge mess. I think its better to raise a separate PR with all the steps followed properly this time. Apologies for any inconvenience caused.

**Pull request closed without merging**